client/app/bundles/shared/components/buttonApp.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+
+class buttonApp extends React.Component {
+   render() {
+       
+       var buttonStyle = {
+         fontSize: 18,
+         color: '#6D0839',
+         fontWeight: "bold",
+         backgroundColor: '#FFFFFF',
+         paddingTop: 17,
+         paddingBottom: 16,
+         paddingLeft: 15,
+         paddingRight: 15,
+         width: 203,
+         borderRadius: 5,
+         borderColor: '#FFFFFF'
+      }
+      return (
+          <div>
+          <ButtonFilledM/>
```

Comment:
Let's separate each of these components you built into separate files! You can place these files under `client/app/bundles/buttons`.